CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ nav_order: 1
 
 - Migrate `aiven_service_integration` to the Plugin Framework
 - Deprecating `project_user`, `account_team` and `account_team_member` resources
+- Improve contributing documentation on running acceptance tests.
```

Comment:
no changelog for docs

Reply:
I skipped this in the first version of the PR, but there's a "Required" check on PRs named "Changelog Enforcer" that fails if there's no changelog entry, even though only the docs were changed, so it looks like "changelog even for docs" is this repo's policy.

Reply:
there's a `no changelog` label, I've added it for your PR now, and this check is now skipped